kratos/utilities/string_utilities.cpp

```diff
@@ -47,6 +45,41 @@ std::string ConvertCammelCaseToSnakeCase(const std::string& rString)
 /***********************************************************************************/
 /***********************************************************************************/
 
+std::string ConvertSnakeCaseToCamelCase(const std::string& rString)
+{
+    std::string output;
+
+    if (!rString.empty()) {
+        output.reserve(rString.size());
+        bool upper_switch = rString[0] == '_' ? false : true;
+
+        for (auto character : rString) {
+            KRATOS_ERROR_IF(!(std::isalnum(character) || character == '_') || std::isupper(character))
+                << "Invalid character '" << character
+                <<"' in snake case string '" << rString << '\'';
+
+            if (character == '_') {
+                KRATOS_ERROR_IF(upper_switch)
+                    << "Repeated underscores in snake case string '" << rString << '\'';
+                upper_switch = true;
+            } else { // character != '_'
+                // At this point, the character must be in [a-z]
```

Comment:
Very minor comment: [0-9] will also go into this flow.

Reply:
ah yes you're right, thanks